android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/file/GlobalUploadsDependencyProvider.kt

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.prompts.file
+
+import androidx.annotation.VisibleForTesting
+import mozilla.components.browser.state.store.BrowserStore
+
+/**
+ * Provides global access to the dependencies needed for purging temporal uploaded files.
+ */
+/** @suppress */
+object GlobalUploadsDependencyProvider {
```

Comment:
I think we can remove this and directly inject `browserStore` instance in `FileUploadsDirCleaner`